lib/mobility/plugins/ransack.rb

```diff
@@ -35,11 +42,14 @@ def result(opts = {})
         private
 
         def apply_mobility_scope(relation, predicate, attributes)
-          (attributes & object.mobility_attributes).inject(relation) do |i18n_rel, attr|
```

Comment:
`mobility_attributes` is now protected, since we shouldn't actually need it anywhere external, and indeed here we can use `mobility_attribute?` instead.